Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml

```
@@ -539,4 +890,357 @@
+      - state: coatybits-inhand-right
+        color: "#3f3f3f"
+      - state: winterbits-inhand-right
+  - type: Clothing
+    sprite: Clothing/OuterClothing/WinterCoats/coat.rsi
+    clothingVisuals:
+      outerClothing:
+      - state: coatybits-equipped-OUTERCLOTHING
+        color: "#1d1d1d"
+      - state: winterbits-equipped-OUTERCLOTHING
+  - type: ToggleableClothing
+    clothingPrototypes: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)